OasisDEX/oasis-borrow
# 🧿 Ajna Multiply position overview

# [Ajna Multiply position overview](https://app.shortcut.com/oazo-apps/story/8665/position-overview-information-ui)

UI for both open and manage Ajna Multiply. So far no data connection, just components filled with static dummy data.

![image](https://user-images.githubusercontent.com/16230404/231431911-6a639d3c-0552-488d-a1cc-8e254179d699.png)

  
## Changes 👷‍♀️

- Created overview section for Ajna Multiply along with its content cards and footer component,
- reorganized Ajna content cards directories - as some of them are reused between different products, I moved them to `common` directory,
- some minor adjustments to details section footer components to accommodate more than three items.
  
## How to test 🧪

See on `/ajna/multiply/WBTC-USDC` - can't tests on existing positions since so far it's not possible to create one.
